documentation/asciidoc/topics/attributes/downstream-attributes.adoc

```diff
@@ -13,6 +13,9 @@
 
 :example_crd_name: infinispan
 :example_namespace: rhdg-namespace
+:subscription_namespace: openshift-operators
+:subscription_name: datagrid
+:operator_source: redhat-operators
```

Comment:
We need to add `operator_channel` here.

Reply:
Hey! We do have another attribute file downstream 
https://gitlab.cee.redhat.com/red-hat-jboss-data-grid-documentation/rhdg-documentation/-/blob/8.4.x/rhdg-common/attributes/rhdg-attributes.adoc?plain=1#L62
It's a bit confusing but for now I'd like to keep it this way. When the downstream docs are build, the attribute is available.
I plan to do a follow-up review of the attributes and document the use of them